workers/unity/Assets/Gdk/Core/Systems/SpatialOSSendSystem.cs

```diff
@@ -19,13 +19,34 @@ internal static class Warnings
         private readonly List<int> registeredReplicators = new List<int>();
         private readonly List<int> commandSenders = new List<int>();
 
+        private int ticks = 0;
+        private const int METRIC_TICKS_UPDATE = 200;
```

Comment:
We use Pascal case for constants. `MetricTicksUpdate`. Also, we should use time to gate metric sends rather than ticks as ticks have a variable length between them.

Reply:
Refactored, no longer an issue!